SADL/AM-CDM-manufacturingProcess.sadl

```diff
@@ -0,0 +1,56 @@
+uri "http://kdl.ge.com/AM-CDM-manufacturingProcess" alias AM-CDM-manufacturingProcess version "$Revision:$ Last modified on   $Date:$".
+
+import "http://kdl.ge.com/AM-CDM-process".
+import "http://kdl.ge.com/AM-CDM-system".
+import "http://kdl.ge.com/AM-CDM-material".
+
+ManufacturingProcessStep (note "Planned process in which input material is transformed or modified") is a ProcessStep,
```

Comment:
The purpose of test is not transform material input. TestProcessStep should be defined seperately.

Reply:
Relates to https://github.com/kaggour/AM-CDM/pull/15#discussion_r1840347086

But shows that the definition would need to be generalized as well as the definition.

Reply:
Based on conversation during 11/20 meeting: TestInspectionCharacterization is a PlannedProcessStep not a ManufacturingProcessStep to allow flexibility and semantic goodness

Also, powder does not have a concept of "design" or "orientation" so a separation between PartTIC and PowderTIC will likely be required.

Alex to make changes.